src/test/scala/io/dockstore/Requests.scala

```diff
@@ -347,6 +357,7 @@ object Requests {
     def getRss = {
       http("Get RSS")
         .get("/metadata/rss")
+        .headers(Map("Accept" -> "text/xml"))
```

Comment:
Oh, and the sitemap endpoint (used only by google indexing right now, but could be stressful)